Configuration/Geometry/python/GeometryExtended_cff.py

```diff
@@ -9,6 +9,7 @@
 # Ideal geometry, needed for simulation
 from Geometry.CMSCommonData.cmsExtendedGeometryXML_cfi import *
 from Geometry.TrackerNumberingBuilder.trackerNumberingGeometry_cfi import *
+from Geometry.TrackerGeometryBuilder.TrackerAdditionalParametersPerDet_cfi import *
```

Comment:
I would like to a warning to readers about why this line is included. I suggest adding a comment along the lines of:
```
# TrackerAdditionalParametersPerDet contains only default values -- needed for consistency with Phase 2
```
If we have to touch Run 2, at least let's leave some explanation.